omegaconf/base.py

```diff
@@ -523,10 +540,10 @@ def _re_parent(self) -> None:
 
 def _parse_assert(condition: Any, msg: str = "") -> None:
```

Comment:
this method does not parse anything, certainly not an assert ;)

maybe `_cond_parse_error(condition: Any, msg: str = "")`?

Reply:
Oops, renamed in 94474ae
I intended it to be understood as "parsing_assert" but yeah, that was a bad name :)